diagnostic.md

```diff
@@ -150,7 +175,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+One will float to the top left, two, three, and four will float to the top right. While five will be on the left hand side under one.
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).